### PR TITLE
October 2024 Newsletter

### PR DESCRIPTION
**Pull request by Robadob**

### CODE REVIEW

File: _newsletters/2024-10-10-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+- [CMI Town Hall](https://www.sheffield.ac.uk/machine-intelligence/events/shefai-community-meeting-8) (Weds 16th Oct) Eighth CMI community and networking event.
+- Open Access Week: Oct 21st-25th is International Open Access Week, and the university library is hosting a [programme of events](https://www.sheffield.ac.uk/library/open-access/open-access-week-2024) to explore the value and possibilities of open access scholarship.
+    - Tues Oct 22nd 1-1.30pm - White Rose University Press - A bitesize introduction. [Book your place here](https://www.eventbrite.co.uk/e/white-rose-university-press-a-bitesize-introduction-tickets-1029731543767)
+- Open Research Conversation's [Autumn-Winter Schedule](https://www.sheffield.ac.uk/openresearch/events/open-research-conversations) has now been announced with 5 Wednesday lunchtime events from October through February.
+    - Weds Oct 23rd 12-1pm - Beyond open access: Democratising knowledge through accessible research outputs. [Book your place here](https://www.eventbrite.co.uk/e/beyond-open-access-democratising-knowledge-through-accessible-outputs-tickets-1028028971327).
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md:34]: Nice way of avoiding duplication between Open Access Week and Open Research Conversations :+1:

File: _newsletters/2024-10-10-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+- [Dual licensing R packages with code and data](https://epiverse-trace.github.io/posts/data-licensing.html)
+- [Technical Properties of Sustainable Software](https://bssw.io/blog_posts/technical-properties-of-sustainable-software)
+
+
+#### Papers
+
+- [2024] [Research Software Engineering: Bridging Knowledge Gaps](https://drops.dagstuhl.de/entities/document/10.4230/DagRep.14.4.42)
+- [2024] [Understanding and effectively mitigating code review anxiety](https://link.springer.com/article/10.1007/s10664-024-10550-9)
+- [2024] [A Vision on Open Science for the Evolution of Software Engineering Research and Practice](https://arxiv.org/abs/2405.12132)
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: I think its useful to mark pre-prints as such as they've not been peer-reviewed...

```suggestion
- [2024] **Pre-print** [A Vision on Open Science for the Evolution of Software Engineering Research and Practice](https://arxiv.org/abs/2405.12132)
```

Could easily go at the end of the line though, no particular style or preference here.

File: _newsletters/2024-10-10-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    - Tues Oct 22nd 1-1.30pm - White Rose University Press - A bitesize introduction. [Book your place here](https://www.eventbrite.co.uk/e/white-rose-university-press-a-bitesize-introduction-tickets-1029731543767)
+- Open Research Conversation's [Autumn-Winter Schedule](https://www.sheffield.ac.uk/openresearch/events/open-research-conversations) has now been announced with 5 Wednesday lunchtime events from October through February.
+    - Weds Oct 23rd 12-1pm - Beyond open access: Democratising knowledge through accessible research outputs. [Book your place here](https://www.eventbrite.co.uk/e/beyond-open-access-democratising-knowledge-through-accessible-outputs-tickets-1028028971327).
+
+#### Upcoming External Events
+
+- [pyOpenSci Fall Festival 2024](https://www.eventbrite.com/e/pyopensci-fall-festival-2024-registration-1000762657047) (Oct 29th - Nov 1st) - A week of events to empower scientists to write better code and share their outcomes openly.
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: @ptheywood has dropped this in late today, date will have passed come the next newsletter...

```suggestion
- [pyOpenSci Fall Festival 2024](https://www.eventbrite.com/e/pyopensci-fall-festival-2024-registration-1000762657047) (Oct 29th - Nov 1st) - A week of events to empower scientists to write better code and share their outcomes openly.
- [DiRAC GPU Hackathon, Durham University](https://dirac.ac.uk/training_events/dirac-nvidia-hackathon-2024/) (Nov 26-28th), application deadline 12pm October 16th. 
```

Adding here so its in chronological order.

Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: Other events have dates before link but realistically doesn't make much difference...

```suggestion
- Oct 29th to Nov 1st - [pyOpenSci Fall Festival 2024](https://www.eventbrite.com/e/pyopensci-fall-festival-2024-registration-1000762657047) - A week of events to empower scientists to write better code and share their outcomes openly.
```

File: _newsletters/2024-10-10-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+- [UK Technology Specialists Network Conference](https://itss.org.uk/uk-technology-specialists-network-conference-2025/) April 29-30th 2025
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: As above following dates first...

```suggestion
- April 29-30th 2025 - [UK Technology Specialists Network Conference](https://itss.org.uk/uk-technology-specialists-network-conference-2025/) 
```

File: _newsletters/2024-10-10-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+#### Articles & Blogs
+
+- [Dual licensing R packages with code and data](https://epiverse-trace.github.io/posts/data-licensing.html)
+- [Technical Properties of Sustainable Software](https://bssw.io/blog_posts/technical-properties-of-sustainable-software)
+
+
+#### Papers
+
+- [2024] [Research Software Engineering: Bridging Knowledge Gaps](https://drops.dagstuhl.de/entities/document/10.4230/DagRep.14.4.42)
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: As with the pre-print making it clear its not peer-reviewed pulication.

```suggestion
- [2024] **Seminar Report** [Research Software Engineering: Bridging Knowledge Gaps](https://drops.dagstuhl.de/entities/document/10.4230/DagRep.14.4.42)
```

File: _newsletters/2024-10-10-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+### Training
+
+#### At Sheffield
+
+**FAIR<sup>2</sup>**
+
+The [FAIR<sup>2</sup> for Research Software Training Programme](https://rse.shef.ac.uk/training/fair4rs/) has been developed
+by the RSE, Data Analytics Service and Library teams and will be launched in October. An introduction to the programme
+will be given in a [LunchBytes session 2024-10-22](../events/lunchbytes-2024-10-22.html). The full syllabus is now live on [MyDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/curriculum/06573f55-12e4-44f1-be4c-fac5ea9d6851) (requires login).
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md:74]: `<pedantic>`Hard wrap to match rest of paragraph`</pedantic>`

```suggestion
will be given in a [LunchBytes session 2024-10-22](../events/lunchbytes-2024-10-22.html). The full syllabus is now live on 
[MyDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/curriculum/06573f55-12e4-44f1-be4c-fac5ea9d6851) (requires login).
```

File: _newsletters/2024-10-10-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+
+**Sheffield Bioinformatics Core**
+
+The Sheffield Bioinformatics Core is pleased to announce a schedule of courses for the new academic year. These courses provide an introduction to the R programming language, Unix environment (required for the use of the University's High Performance Computing facilities) and also specific applications in RNA-sequencing.
+You can now sign-up for these courses using myDevelopment (search for "bioinformatics"). Course charges still apply, and upon sign-up we will contact you for a valid grant code or cost centre. In-person attendance is encouraged, but online options are also available.
+
+- Data Manipulation and Visualization - October 28th, 30th and November 1st 13:00 - 16:00 - (£72 for students and £80 for staff)
+- Introduction to RNA-seq analysis - November 12th 13:00 - 16:00 (£32 for students and £40 for staff)
+Introduction to the Command-Line for Bioinformatics - November 27th 13:00 - 16:00 (£32 for students and £80 for staff)
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]:
```suggestion
- Introduction to the Command-Line for Bioinformatics - November 27th 13:00 - 16:00 (£32 for students and £80 for staff)
```

File: _newsletters/2024-10-10-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+- Introduction to RNA-seq analysis in R - December 1st, 3rd and 5th 13:00 - 16:00 (£72 for students and £80 for staff)
+
+Where multiple dates are listed, you will need to attend all dates. You can also find more information on these course on their website (https://sbc.shef.ac.uk)
+
+#### Outside Sheffield
+
+- [SSI Software Practices in the Social Sciences programme](https://www.software.ac.uk/blog/ssi-announces-research-software-practices-social-sciences-programme) - A new programme of 1-hour workshops covering topics such as Relational Databases, Reproducible Analysis and Data Cleanup for Social Scientists.
+- [UK Data Service](https://ukdataservice.ac.uk/) provide access to a wide range of reusable social sciene data, and training to enable researchers to make the most of it.
+    - Mon Oct 14th 10am - [Introduction to synthetic data for longitudinal data managers](https://ukdataservice.ac.uk/events/introduction-to-synthetic-data-for-longitudinal-data-managers/)
+    - Mon 10th 10am - [Safe Researcher Training](https://ukdataservice.ac.uk/events/safe-researcher-training-67/)
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: Link says its 14th Oct

```suggestion
    - Mon Oct 14th 10am - [Safe Researcher Training](https://ukdataservice.ac.uk/events/safe-researcher-training-67/)
```

File: _newsletters/2024-10-10-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+    - with recordings of past trainings available on their [YouTube](https://www.youtube.com/ukdataservice)
+
+### Funding Opportunities
+
+- [FAIR-IMPACT's 3rd call](https://fair-impact.eu/3rd-open-call-route-2-support-opens-30-sept), deadline 4th Dec, hopes to fund applicants in four areas
+    - Managing data types, schemas and vocabularies and crosswalks for FAIR researcher related metadata (in partnership with FAIRCORE4EOSC)
+    - Creating EOSC compliant interoperability policies based on the EOSC Interoperability Framework (IF)
+    - Testing the trustworthy and FAIR-enabling repositories prototype
+    - Implementation of a shared API for semantic catalogues
+#### Jobs
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: `<pedantic>`Linting would suggest spacing`</pedantic>`

```suggestion

#### Jobs
```

File: _newsletters/2024-10-10-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+
+There are two positions within the University of Sheffield currently being advertised.
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]:
```suggestion
```

File: _newsletters/2024-10-10-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+
+-
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]:
```suggestion
```

File: _newsletters/2024-10-10-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
+#### Digital Research Practice Support Community
+
+The DRPS community is a group for people that support researchers in carrying out research in the digital age. Meetings
+are held monthly, with discussions around events, training and opportunities related to the field.
+
+You can join the google group
+[here](https://groups.google.com/u/1/a/sheffield.ac.uk/g/digital-research-practice-support-community-group/about) to
+stay informed.
+
+The next meeting is scheduled for 2pm on Wednesday 18th September 2024.
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]:
```suggestion
The next meeting is scheduled for 2pm on Wednesday 6th November 2024.
```

File: _newsletters/2024-10-10-newsletter.md [hunk 13]
```diff
@@ -0,0 +1,174 @@
...
+#### LunchBytes
+
+[LunchBytes](https://rse.shef.ac.uk/community/lunch-bytes/) are short talks from the research community on research
+software, data, and infrastructure.
+
+##### LunchBytes needs YOU
+
+LunchBytes are organised by and for the research software community at The University of Sheffield. If you'd like to
+curate a session on a topic or present something, get in touch by emailing
+[lunchbytes-organisers-group@sheffield.ac.uk](mailto:lunchbytes-organisers-group@sheffield.ac.uk) or suggest topics
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]: Was mentioned today that Farhad will be taking over organising these, we should make sure that he's on this mailing list.

I've tried joining [https://groups.google.com/a/sheffield.ac.uk/g/lunchbytes-organisers-group](https://groups.google.com/a/sheffield.ac.uk/g/lunchbytes-organisers-group) but without success, page isn't found.

File: _newsletters/2024-10-10-newsletter.md [hunk 14]
```diff
@@ -0,0 +1,174 @@
...
+
+**FAIR<sup>2</sup>**
+
+The [FAIR<sup>2</sup> for Research Software Training Programme](https://rse.shef.ac.uk/training/fair4rs/) has been developed
+by the RSE, Data Analytics Service and Library teams and will be launched in October. An introduction to the programme
+will be given in a [LunchBytes session 2024-10-22](../events/lunchbytes-2024-10-22.html). The full syllabus is now live on [MyDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/curriculum/06573f55-12e4-44f1-be4c-fac5ea9d6851) (requires login).
+
+**Sheffield Bioinformatics Core**
+
+The Sheffield Bioinformatics Core is pleased to announce a schedule of courses for the new academic year. These courses provide an introduction to the R programming language, Unix environment (required for the use of the University's High Performance Computing facilities) and also specific applications in RNA-sequencing.
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md:78]: Pedantic line wrapping...

```suggestion
The Sheffield Bioinformatics Core is pleased to announce a schedule of courses for the new academic year. These courses 
provide an introduction to the R programming language, UNIX environment (required for the use of the University's High
 Performance Computing facilities) and also specific applications in RNA-sequencing.
```

File: _newsletters/2024-10-10-newsletter.md [hunk 15]
```diff
@@ -0,0 +1,174 @@
...
+You can now sign-up for these courses using myDevelopment (search for "bioinformatics"). Course charges still apply, and upon sign-up we will contact you for a valid grant code or cost centre. In-person attendance is encouraged, but online options are also available.
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md:79]: Pedantic line wrapping...

```suggestion
You can now sign-up for these courses using myDevelopment (search for "bioinformatics"). Course charges still apply, and 
upon sign-up we will contact you for a valid grant code or cost centre. In-person attendance is encouraged, but 
online options are also available.
```

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for knocking this up so quickly @Robadob especially with how busy you've been the last few weeks :+1: 

Bunch of minor things, many pedantic and can be safely ignored. Some more important such as a couple of dates seemed to be out.

File: _newsletters/2024-10-10-newsletter.md (hunk 1 above)
Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md:34]: I just put the next event from each, did notice the overlap.

File: _newsletters/2024-10-10-newsletter.md (hunk 12 above)
Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md]: whoops

File: _newsletters/2024-10-10-newsletter.md (hunk 13 above)
Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md]: Yeah, I wasn't sure how to phrase that given nothing yet in the pipeline.

Not sure about the missing group, maybe wilby owned it so it got deleted?

File: _newsletters/2024-10-10-newsletter.md (hunk 14 above)
Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md:78]: my text editor supports word-wrap, waste of screen space if it's force wrapped different to my editor window..

**Comment by Robadob** [comment]
Need to review date formatting when I have more attention span left, otherwise think things are addressed. + lunchbytes (have messaged @f-allian on our slack).

File: _newsletters/2024-10-10-newsletter.md (hunk 6 above)
Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md:74]: image
It looks fine in my text editor without any changes 🤷‍♂️

Comment (ns-rse) [reply _newsletters/2024-10-10-newsletter.md:74]: My pedantry with line wraps is less to do with how it appears in editors which will do soft line-wraps, and in Emacs you get an indicator that these are soft-wraps with little arrows at the side...

soft-wrapped-lines

...rather when it comes to line diffing and finding errors/rendering differences its much harder to see these in really long chunks of text. That is somewhat ameliorated with the `--word-diff=color` but I've just got a old habit of hard-wrapping lines (well not so much a habit, it just happens automatically in my editor or its done by linters).

As I say, its pedantic and doesn't matter as it doesn't affect the rendered content which is wrapped automatically based on the available space.

File: _newsletters/2024-10-10-newsletter.md (hunk 10 above)
Comment (ns-rse) [reply _newsletters/2024-10-10-newsletter.md]: Not currently positions being advertised and there was the announcement of a moratorium on recruitment.

Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md]: I think you're reviewing an old version this should be gone.

File: _newsletters/2024-10-10-newsletter.md (hunk 6 above)
Comment (Robadob) [reply _newsletters/2024-10-10-newsletter.md:74]: Good diff checkers typically highlight affected characters within a line if tokens have changed but most of the line remains the same.

e.g.
image

or

image

File: _newsletters/2024-10-10-newsletter.md [hunk 16]
```diff
@@ -0,0 +1,171 @@
...
+    - Tues Oct 22nd 1-1.30pm - White Rose University Press - A bitesize introduction. [Book your place here](https://www.eventbrite.co.uk/e/white-rose-university-press-a-bitesize-introduction-tickets-1029731543767)
+- Open Research Conversation's [Autumn-Winter Schedule](https://www.sheffield.ac.uk/openresearch/events/open-research-conversations) has now been announced with 5 Wednesday lunchtime events from October through February.
+    - Weds Oct 23rd 12-1pm - Beyond open access: Democratising knowledge through accessible research outputs. [Book your place here](https://www.eventbrite.co.uk/e/beyond-open-access-democratising-knowledge-through-accessible-outputs-tickets-1028028971327).
+
+#### Upcoming External Events
+
+- Until Nov 10th - [MATLAB Shorts Mini Hack](https://uk.mathworks.com/matlabcentral/contests/2024-matlab-mini-hack.html): Use upto 2,000 characters ofr MATLAB code to generated a MATLAB movie and win prizes.
```
Comment (ns-rse) [inline _newsletters/2024-10-10-newsletter.md]:
```suggestion
- Until Nov 10th - [MATLAB Shorts Mini Hack](https://uk.mathworks.com/matlabcentral/contests/2024-matlab-mini-hack.html): Use upto 2,000 characters of MATLAB code to generate a MATLAB movie and win prizes.
```

File: _newsletters/2024-10-10-newsletter.md (hunk 10 above)
Comment (ns-rse) [reply _newsletters/2024-10-10-newsletter.md]: Indeed, apologies, I thought GitHub kept things fresh but it seems not.

**Pull request merged**